### ISSUE TITLE
get_profile_connections

### ISSUE THREAD

**Issue by sambhavjain9138** [tomquirk/linkedin-api#163]
I have 4000+ connections on linkedin, but when i try fetching result using this function,, it returns me only 997 connections. I tried setting offset to check if this limitation is from linkedin's side, but found that it still returns me 997-offset results.

Can anybody help me on how to fetch the whole list?

**Comment by ReddyKatta89** [comment]
Hi Sambhav,

Were you able to fix this issue?

**Comment by sambhavjain9138** [comment]
Heyy Sir
No, I wasn't able to fix the issue. Its still there. Are you too facing the same issue?

**Comment by alvaroserrrano** [comment]
I believe that endpoint returns only first-degree connections, so it would only return the profiles of the people that both the logged-in user and the target user have in common. 
Have there been any updates? Are you aware of any endpoint that returns all the connections of a profile even if you have no relationship with that profile whatsoever?

**Comment by abinpaul1** [comment]
For the case of endpoint to get connections of a profile you have no relationship with, I don't think Linkedin provides that functionality.

**Comment by igorgbianchi** [comment]
If you explore your profile connections, you'll see that Linkedin shows only 1000 connections and I think this package uses the same API as Linkedin's frontend.

**Comment by abinpaul1** [comment]
Yes, @igorgbianchi that would explain it. This package does indeed use the same Api as Linkedin website
